web-transport-proto/src/capsule.rs

```diff
@@ -107,7 +107,10 @@ fn is_grease(val: u64) -> bool {
     if val < 0x21 {
         return false;
     }
-    (val - 0x21) % 0x1f == 0
+    #[allow(clippy::manual_is_multiple_of)]
+    {
+        (val - 0x21) % 0x1f == 0
```

Comment:
```rust
(val - 0x21).is_multiple_of(0x1f)
``` 

I guess but I don't care.

Reply:
This would force raising MSRV to Rust 1.87